packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart

```diff
@@ -232,24 +236,24 @@ class ImagePickerPlugin extends ImagePickerPlatform {
   /// Handles the OnChange event from a FileUploadInputElement object
   /// Returns a list of selected files.
   List<html.File>? _handleOnChangeEvent(html.Event event) {
-    final html.FileUploadInputElement input =
-        event.target as html.FileUploadInputElement;
-    return _getFilesFromInput(input);
+    final html.FileUploadInputElement? input =
+        event.target as html.FileUploadInputElement?;
+    return input == null ? null : _getFilesFromInput(input);
```

Comment:
I'm not aware that input here can be null (html.Event has a nullable target because you can create your own custom events without setting it up, but AFAIK the DOM Events should always contain a target); another acceptable solution would have been:

```dart
return _getFilesFromInput(input!);
```

Reply:
I did this based on the return type being nullable. Maybe that was wrong too? But all the callers already have handling for it, so if this should be force-unwrapped there's cleanup that should happen throughout this file.

Reply:
Nope, this is good. The return type is nullable because the behavior in general is nullable (except in this particular case (and maybe others) where the event comes from the browser, and it will always fill the `target`).